portfolio/src/main/java/com/google/sps/servlets/StringServlet.java

```diff
@@ -19,9 +19,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //create a new array list and input quoataions.
```

Comment:
Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace